scripts/new.js

```diff
@@ -6,6 +6,8 @@ if (localStorage.getItem('books') === null) {
   books = JSON.parse(localStorage.getItem('books'));
 }
 
+// localStorage.clear();
+
```

Comment:
- [ ] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)

Reply:
## Resolved 👌 

- that was a used for debugging
- used `localStorage.clear() ` to clean and retest
- removed ASAP
- thanks for the assist
![Horizon Node](https://media.giphy.com/media/o2LivNgy8GYqQbDdYx/giphy.gif)